packages/next/client/index.tsx

```diff
@@ -80,6 +88,17 @@ if (hasBasePath(asPath)) {
   asPath = delBasePath(asPath)
 }
 
+asPath = delLocale(asPath, locale)
```

Comment:
Should this consider basePath as well?

Reply:
The `basePath` should be removed by this point so it shouldn't need to worry about it being present when removing the locale here